app/code/community/Mbiz/TrackingTags/Model/Filter.php

```diff
@@ -39,7 +42,16 @@ public function productDirective($construction)
         }
 
         if (is_object($product) && $product->getId()) {
+            // Fire event to implement custom behaviour if needed, the object can be modified by the event
+            $object = new Varien_Object();
             $params   = $this->_getIncludeParameters($construction[2]);
+            Mage::dispatchEvent('mbiz_trackingtags_product_directive', array('object' => $object, 'params' => $params, 'product' => $product));
```

Comment:
Please use short array syntax.

And you can jump some lines to get a better reading experience :')